cfpb/cfgov-refresh
# Update expandables to use CF version

Working on replacing the cfgov specific expandables with the CF expandables

## Additions

- Adds latest cf-expandables to package.json

## Removals

- Removes individual instances of expandables code from the project

## Changes

- Updated modifications to default expandables where necessary to work with cf-expandables

## Testing

1. Run `npm install && gulp clean && gulp build`
1. Navigate to http://localhost:8000/about-us/the-bureau/bureau-structure/ and use the expandables within the Bureau Structure tree.
1. Reduce the screen size and use the secondary nav expandable
1. Navigate to http://localhost:8000/consumer-tools/money-as-you-grow/young-adulthood/, at the small screen sizes the secondary nav should not be visible
1. Expand the screen size and use the expandables within the content
1. Navigate to http://localhost:8000/owning-a-home/closing-disclosure/ and use the expandables.
1. Navigate to http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/ and use the expandable in the answer content
1. Navigate to http://localhost:8000/consumer-tools/financial-well-being/ and fill in the answers, then submit the form, use the expandable on the following page.
1. Navigate to http://localhost:8000/about-us/blog/ and open the filter expandable
1. Apply a filter and the following page should open with the filter expanded by default

## Screenshots

<img width="391" alt="screen shot 2018-07-24 at 9 38 18 am" src="https://user-images.githubusercontent.com/1280430/43145681-5358b29e-8f25-11e8-8450-c8fca603ecef.png">

<img width="639" alt="screen shot 2018-07-24 at 9 38 37 am" src="https://user-images.githubusercontent.com/1280430/43145713-5da32a0e-8f25-11e8-9e8e-69717e126280.png">

<img width="693" alt="screen shot 2018-07-24 at 9 39 34 am" src="https://user-images.githubusercontent.com/1280430/43145825-812799f6-8f25-11e8-84ad-a43bc2ef4a4c.png">

<img width="913" alt="screen shot 2018-07-24 at 9 40 04 am" src="https://user-images.githubusercontent.com/1280430/43145887-925dae2c-8f25-11e8-9899-1761530f9a3b.png">

<img width="620" alt="screen shot 2018-07-24 at 9 40 36 am" src="https://user-images.githubusercontent.com/1280430/43145924-a5b90eda-8f25-11e8-867f-6f0b56932f92.png">

<img width="673" alt="screen shot 2018-07-24 at 9 41 17 am" src="https://user-images.githubusercontent.com/1280430/43145962-bf34370e-8f25-11e8-90b3-eeccf8682a05.png">

<img width="673" alt="screen shot 2018-07-24 at 9 41 51 am" src="https://user-images.githubusercontent.com/1280430/43145994-d1b44ee6-8f25-11e8-892b-eed06fa63646.png">


## Notes

- Some instances of expandables should possibly not be expandables (secondary nav) or should use a default expandable pattern(bureau structure), or should have their modifications ported back to CF (read more)

## Todos

- Update cf-expandables for necessary event emitters and interaction methods

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
